gulpfile.js

```diff
@@ -21,6 +21,7 @@ var gulp = require('gulp'),
     connectModRewrite = require('connect-modrewrite'),
     ngConfig = require('gulp-ng-config'),
     prettyError = require('gulp-prettyerror'),
+    path = require('path'),
```

Comment:
But then how it gets install, actually every gulp module is installed using npm, and I can see that path is not included in package.json. Is it still working fine?